packages/webapp/webapp_server.js

```diff
@@ -868,7 +868,12 @@ function runWebAppServer() {
         onListeningCallbacks.push(f);
       else
         f();
-    }
+    },
+    // This can be overridden by users who want to modify how listening works
+    // (eg, to run a proxy like Apollo Engine Proxy in front of the server).
+    startListening: function(httpServer, listenOptions, cb) {
```

Comment:
Super nit-picky code formatting comment - most of the anonymous functions in this file have a space after `function` (by no means a show-stopper 🙂).

Reply:
Sorry to add to the nit-pickyness, but there should also be an empty line before a comment, according to the guide.